mcda/mcda_run.py

```diff
@@ -1,377 +1,165 @@
 #! /usr/bin/env python3
 
-import logging
+"""
+This script serves as the main entry point for running all pieces of functionality in a consequential way by
+following the settings given in the configuration file 'configuration.json'.
+
+Usage (from root directory):
+    $ python3 -m mcda.mcda_run -c configuration.json
+"""
+
 import time
-import sys
-import os
+import logging
 
 from mcda.configuration.config import Config
-from mcda.utils import *
-from mcda.utils_for_parallelization import *
-from mcda.mcda_without_robustness import MCDAWithoutRobustness
-from mcda.mcda_with_robustness import MCDAWithRobustness
+from mcda.utils.utils_for_main import *
+from mcda.utils.utils_for_plotting import *
+from mcda.utils.utils_for_parallelization import *
+
+log = logging.getLogger(__name__)
 
-formatter = '%(levelname)s: %(asctime)s - %(name)s - %(message)s'
-logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format=formatter)
+FORMATTER: str = '%(levelname)s: %(asctime)s - %(name)s - %(message)s'
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format=FORMATTER)
 logger = logging.getLogger("ProMCDA")
 
-class UserStoppedInfo(Exception):
-    pass
+# noinspection PyTypeChecker
+def main(input_config: dict):
+    """
```

Comment:
Indentation is off for `"""`

Reply:
Fixed at [0ea183a](https://github.com/wetransform-os/ProMCDA/commit/0ea183a2db9ba6320492e729fb4fb18c3b98ea71)